src/components/Restaurants/RestaurantReview.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FaSearch } from 'react-icons/fa'; 
+import './RestaurantList.css';
+
+const SearchBar = () => (
+  <header className="header">
+    <div className="searchContainer">
+      <input
+        type="text"
+        placeholder="Restaurants"
+        className="searchInput"
+        aria-label="Search for restaurants"
+      />
+      <input
+        type="text"
+        placeholder="Address, neighborhood"
+        className="locationInput"
+        aria-label="Location"
+      />
+      <button className="searchButton">
+        <FaSearch />
+      </button>
+    </div>
+  </header>
+);
+
+const Filters = () => (
+  <div className="filters">
+    <div className="filterSection">
+      <button className="button">$</button>
+      <button className="button">$$</button>
+      <button className="button">$$$</button>
+      <button className="button">$$$$</button>
+    </div>
+    <div className="filterSection">
+      <label className="checkboxLabel">
+        <input type="checkbox" className="checkbox" />
+        Open Now
+      </label>
+      <label className="checkboxLabel">
+        <input type="checkbox" className="checkbox" />
+        Reservations
+      </label>
+    </div>
+    <div className="filterSection">
+      <button className="categoryButton">Mexican</button>
+      <button className="categoryButton">Sandwiches</button>
+    </div>
+  </div>
+);
+
+const RestaurantEntry = ({ imageSrc, name, tags, rating, reviewCount, url }) => (
+  <div className="restaurantEntry">
+    <img src={imageSrc} alt={name} className="restaurantImage" />
+    <div className="restaurantDetails">
+      <a href={url} className="restaurantName">{name}</a>
+      <div className="restaurantTags">{tags.join(' · ')}</div>
```

Comment:
Instead of "join", which will make it 1 single string, each tag should be its own div. Use `tags.map()` here. Create a div outside the map for the container

Notes on the css of the tags:
- the tag should be on 1 line (container can be `div: flex`)
- background color: rgba(240, 240, 240, 1)
- font-size: 12px
- border-radius: ~4px
- padding: ~3px
- color: rgba(110,112,114,1)